standalone-metastore/metastore-server/src/main/sql/hive/upgrade-4.0.0-to-4.1.0.hive.sql

```diff
@@ -1,3 +1,10 @@
 SELECT 'Upgrading MetaStore schema from 4.0.0 to 4.1.0';
 
+USE SYS;
+
+DROP TABLE IF EXISTS `VERSION`;
+
+CREATE OR REPLACE VIEW `VERSION` AS SELECT 1 AS `VER_ID`, '4.1.0' AS `SCHEMA_VERSION`,
+  'Hive release version 4.1.0' AS `VERSION_COMMENT`;
```

Comment:
We need to upgrade this definition. Otherwise, the final validation fails.

```
Completed upgrade-4.0.0-to-4.1.0.hive.sql
2024-06-27T03:05:43,358 ERROR [main] schematool.MetastoreSchemaTool: Metastore schema version is not compatible. Hive Version: 4.1.0, Database Schema Version: 4.0.0-alpha-1
Metastore schema version is not compatible. Hive Version: 4.1.0, Database Schema Version: 4.0.0-alpha-1
2024-06-27T03:05:43,358 ERROR [main] schematool.MetastoreSchemaTool: Use --verbose for detailed stacktrace.
Use --verbose for detailed stacktrace.
2024-06-27T03:05:43,358 ERROR [main] schematool.MetastoreSchemaTool: *** schemaTool failed ***
```